shared/reducers/tracker.js

```diff
@@ -457,6 +435,14 @@ function remoteProofToProof (rp: RemoteProof, lcr: ?LinkCheckResult): Proof {
     statusMeta = proofStatusToSimpleProofMeta(lcr.proofResult.status)
   }
 
+  const isTracked = !!(lcr && lcr.diff && lcr.diff.type === identify.TrackDiffType.none && !lcr.breaksTracking)
+
+  // whatevz, tracked a broken thing
+  if (lcr && lcr.proofResult && lcr.proofResult.status !== identify.ProofStatus.ok && isTracked) {
+    diffMeta = metaIgnored
+    statusMeta = null
```

Comment:
you track them, therefore you have a lcr.proofResult. The status isn't ok, so something is wrong. You currently track it, even though something isn't ok. hence, you've ignored it.

to test this, edit your /etc/hosts for a website and break someone you track (i use max's oneshallpass.com for this). You'll see his proof is broken, if you then ignore for 24 hours or retrack you'll see the ignored meta tag